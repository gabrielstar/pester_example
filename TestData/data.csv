header1,header2,header3,header4
1,2,3,4
5,6,7,8