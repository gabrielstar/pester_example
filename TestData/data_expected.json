[
    {
        "header1":  "1",
        "header2":  "2",
        "header3":  "3",
        "header4":  "4"
    },
    {
        "header1":  "5",
        "header2":  "6",
        "header3":  "7",
        "header4":  "8"
    }
]
